src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -531,4 +531,13 @@ default boolean canSee(Entity entity) {
         Optional<Boolean> optional = entity.get(Keys.VANISH);
         return !optional.isPresent() || !optional.get();
     }
+
+    /**
+     * Returns whether this entity has gravity.
+     *
+     * @return True if this entity has gravity
+     */
+    default boolean hasGravity() {
```

Comment:
Also, pretty sure it's meant to follow the Value<> pattern - https://github.com/SpongePowered/SpongeAPI/blob/bleeding/src/main/java/org/spongepowered/api/entity/living/monster/Creeper.java#L36